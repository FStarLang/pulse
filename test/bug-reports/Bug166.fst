module Bug166
open Pulse
#lang-pulse

inline_for_extraction noextract
fn h ()
  requires emp
  returns  x : int
  ensures  emp
{
  42
}
let h2 = h