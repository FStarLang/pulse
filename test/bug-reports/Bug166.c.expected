/* krml header omitted for test repeatability */


#include "Bug166.h"

krml_checked_int_t Bug166_h2(void)
{
  return (krml_checked_int_t)42;
}

